jdaviz/configs/default/plugins/line_lists/line_lists.py

```diff
@@ -23,7 +23,7 @@
 
 
 @tray_registry('g-line-list', label="Line Lists")
-class LineListTool(TemplateMixin):
+class LineListTool(PluginTemplateMixin):
```

Comment:
@kecnry , do I have to undo this? Was this only for POC illustration and not meant for production?